components/validateur-report/numero-row.js

```diff
@@ -0,0 +1,63 @@
+import {useState, useMemo} from 'react'
+import PropTypes from 'prop-types'
+import {Pane, Text, UnorderedList, ListItem} from 'evergreen-ui'
+import {filter} from 'lodash'
+import {getLabel} from '@ban-team/validateur-bal'
+
+import AlertHeader from '@/components/validateur-report/alerts-header'
+import Dropdown from '@/components/dropdown'
+
+function NumeroRow({numero}) {
+  const {address, alerts} = numero
+  const [isNumeroOpen, setIsNumeroOpen] = useState(false)
+
+  const [hasWarnings, hasErrors, hasInfos] = useMemo(() => {
+    return [
+      filter(numero.alerts, ({level}) => level === 'W').length > 0,
+      filter(numero.alerts, ({level}) => level === 'E').length > 0,
+      filter(numero.alerts, ({level}) => level === 'I').length > 0
+    ]
+  }, [numero])
```

Comment:
Pour éviter de réécrire les variables, on peut aussi faire : 
```suggestion
  const alerts = useMemo(() => {
    return {
      hasWarnings: filter(numero.alerts, ({level}) => level === 'W').length > 0,
      hasErrors: filter(numero.alerts, ({level}) => level === 'E').length > 0,
      haseInfos: filter(numero.alerts, ({level}) => level === 'I').length > 0
    }
  }, [numero])
```

De cette façon, on peut déstructurer pour le composant 
```<AlertHeader type='secondary' {...alerts} />```

C'est un simple commentaire et non une demande de changement.